scripts/exgdas_enkf_fcst.sh

```diff
@@ -112,6 +112,10 @@ export FHOUT=${FHOUT_ENKF:-3}
 # model_configure
 export DELTIM=${DELTIM_ENKF:-${DELTIM:-225}}
 export FHMAX=${FHMAX_ENKF:-9}
+if [[ $CDUMP == "gfs" ]]; then
+   export FHMAXF=$FHMAX_ENKF_GFS
```

Comment:
There is a typo here on the LHS.
Also, suggest this:
```suggestion
   export FHMAX=${FHMAX_ENKF_GFS:-${FHMAX_ENKF:-${FHMAX}}}
```